src/csc439teamllama/cardgame/GolfCLIView.java

```diff
@@ -119,15 +117,24 @@ protected void SendMessageToPlayer(String message) {
      * @param game
      */
     @Override
-    protected void DisplayHand(GolfGameModel game) {
-        System.out.println("1."+game.players[game.PlayerIndex()].hand[0].toString()+ " 2." + game.players[game.PlayerIndex()].hand[1].toString()+ " 3." + game.players[game.PlayerIndex()].hand[2].toString());
-        System.out.println("4."+game.players[game.PlayerIndex()].hand[3].toString()+ " 5." + game.players[game.PlayerIndex()].hand[4].toString()+ " 6." + game.players[game.PlayerIndex()].hand[5].toString());
+    protected void displayHand(GolfGameModel game) {
+        System.out.println("1."+game.players[game.playerIndex()].hand[0].toString()+ " 2." + game.players[game.playerIndex()].hand[1].toString()+ " 3." + game.players[game.playerIndex()].hand[2].toString());
+        System.out.println("4."+game.players[game.playerIndex()].hand[3].toString()+ " 5." + game.players[game.playerIndex()].hand[4].toString()+ " 6." + game.players[game.playerIndex()].hand[5].toString());
     }
 
-    /**
-     *allow user to re-input decision
-     */
-    protected void ClearScanner(){
+    @Override
+    protected void displayScoreBoard(GolfGameModel game) {
+        System.out.println("-----SCOREBOARD-----");
+
+        for (int i = 0; i<game.players.length; i++){
```

Comment:
This should actually loop over an array of players we called scoreboard, although we can have a conversation about naming convention but this would just print an array of players in turn order.

Reply:
looks like a good fix